prompb/io/prometheus/write/v2/types.proto

```diff
@@ -49,6 +49,10 @@ message Request {
   repeated string symbols = 4;
   // timeseries represents an array of distinct series with 0 or more samples.
   repeated TimeSeries timeseries = 5 [(gogoproto.nullable) = false];
+  // If true, metric names and labels can use the full UTF-8 character set.
+  // Otherwise, apply the legacy naming convention.
+  // XXXX do we even need this?  content negotiation should take care of it
+  bool utf8_names = 6;
```

Comment:
pretty sure I can drop this completely but worth checking

Reply:
This ideally has to be discussed in PRW 2.0 forum ideally, we want to make sure the spec is  and this is source of truth 🙃 Do we need this? cc @cstyan

Reply:
In the spec rw2.0 we already allow for utf8 characters in names and values, and just have `SHOULD` for the current prometheus name format. Don't think we need this in the proto for 2.0.

Reply:
Do we need this for remote write? Can we parse everything and reject series based on the feature flag?

Reply:
the content negotiation should resolve whether the metrics will be sent as UTF-8 or escaped.  This would cover the case where the remote server *could* support UTF-8 but has that feature turned off for some reason, *and* the sender has UTF-8 metrics it is trying to send.

Alternatively we could just have the remote write fail validation and reject the incoming series